src/android/aidl/client.md

```diff
@@ -20,5 +20,8 @@ Build, push, and run the client on your device:
 
 ```shell
 {{#include ../build_all.sh:birthday_client}}
+```
+
+```shell
```

Comment:
Nice! Could you remove the `shell` language here and elsewhere — the output is just text and a stray `$` should not be syntax highlighted.

Reply:
Sorry, I only saw [your comment now](https://github.com/google/comprehensive-rust/issues/334#issuecomment-1631580582) — please change `shell` to `text`. The default language is `rust` and so `mdbook test` will try to treat the code block as Rust code if you don't explicitly make it `text`.

Reply:
Done ✌️